chore(i18n): use english as the base language for i18n

413x1nkp opened the pull request:
rewrite all of the locale files to use english as base for translation
this PR is a response to the suggestion at #20

i also thought about replacing some keys with keywords, so like
```json
    "ID of model A (long)": "ID of model A (long)",
    "Weight (w) for Model A": "Weight (w) for Model A",
    "Path to Model A": "Path to Model A",
    "ID of model B (long)": "ID of model B (long)",
```
will be
```json
    "ID_mA_L": "ID of model A (long)",
    "W_mA": "Weight (w) for Model A",
    "PATH_mA": "Path to Model A",
    "ID_mB_L": "ID of model B (long)",
```
and in code:
```py
i18n("ID_mA_L")
i18n("W_mA")
# . . .
```
but i thought i should change base language to english first and then do the string-id thing

413x1nkp committed:
chore(i18n): use english as the base language for i18n

rewrite all of the locale files to use english as base for translation

413x1nkp committed:
fix(i18n): update rest of the scripts that rely on the chinese-base i18n translation

413x1nkp commented:
please test who can! although i assume i made no mistakes, it's still possible i missed some of the values (since there are SO many of them)
all tests i've done (mainly tested chinese and english) seem to be working as intended, however i can't test this myself fully, since changing locale on linux is a bit unconventional

fumiama commented on gui.py:
```diff
@@ -358,7 +358,7 @@ def launcher(self):
                             ],
                             [
-                                sg.Text(i18n("共振偏移")),
+                                sg.Text(i18n("Resonating offset")),
```
Well, this translation can be set to the term **Formant offset**.

413x1nkp replied:
> Well, this translation can be set to the term **Formant offset**.

I didn't actually change any of the strings, that's how all of them were in the en_US.json locale file
I can change it though, if You insist

413x1nkp replied:
changed `Resonating offset` to `Formant offset` in `2edfea8`

fumiama replied:
> I didn't actually change any of the strings

I know, but just point out.

fumiama commented on gui.py:
```diff
@@ -380,7 +380,7 @@ def launcher(self):
                             ],
                             [
-                                sg.Text(i18n("响度因子")),
+                                sg.Text(i18n("loudness factor")),
```
cap `L`

413x1nkp replied:
changed in `2edfea8`

fumiama reviewed (commented):
Have you run the `Generate and Sync Locale` action on your fork? You can trigger it by renaming your `strings-xml` branch to `dev`. That workflow can help you detect all the i18ns so that you can check the missing.

413x1nkp committed:
chore(i18n): change some of the base strings to be more correct

413x1nkp commented:
> Have you run the `Generate and Sync Locale` action on your fork? You can trigger it by renaming your `strings-xml` branch to `dev`. That workflow can help you detect all the i18ns so that you can check the missing.

I thought about it, but I'm not sure if the PR would have to be re-made from scratch if i renamed the branch locally and then just removed the original branch at my fork
I'm not really proficient in Git, sorry

fumiama commented:
> but I'm not sure if the PR would have to be re-made from scratch if i renamed the branch

You can delete the `dev` branch at web and rename this branch to dev. This PR will be automatically redirect to dev.

github-actions[bot] committed:
chore(i18n): sync locale on dev

413x1nkp committed:
Merge pull request #2 from alexlnkp/genlocale-dev

chore(i18n): sync locale on dev

github-actions[bot] committed:
chore(format): run black on dev

413x1nkp commented:
Renamed to `dev`, accepted actions' PR, renamed back to `strings-xml`
Truth be told, I didn't even know this action existed... I wrote my own script to just convert all of the locales... lol

413x1nkp committed:
Merge branch 'dev' into strings-xml

413x1nkp committed:
Merge pull request #3 from alexlnkp/formatter-dev

chore(format): run black on dev

413x1nkp committed:
Merge branch 'dev' into strings-xml

fumiama commented:
> Truth be told, I didn't even know this action existed...

Don't worry. There's why I'm here.

fumiama commented:
Seems that you have changed some translations after running the sync-locale. I have fixed ZH_CN and JA_JP that maybe difficult for you to recognize. You can fix the remaining by comparing dev branch with the main branch.

413x1nkp commented:
> Seems that you have changed some translations after running the sync-locale. I have fixed ZH_CN and JA_JP that maybe difficult for you to recognize. You can fix the remaining by comparing dev branch with the main branch.

I'll look into it! Thanks
